form.yml

```diff
@@ -0,0 +1,329 @@
+netweaver:
+  $name: Netweaver
+  $type: group
+  
+  install_packages:
+    $name: Install required packages
+    $type: boolean
+    $default: true
+    $help: Install all required packages to install SAP Netweaver
+  
+  virtual_addresses:
```

Comment:
Add a `help` entry. Explain that this is a `key/value` list or map with the IP addresses and hostnames that later will be used in the installation. That it must contain all the values that will belong to the cluster